main.js

```diff
@@ -524,7 +524,12 @@ app.on('ready', function() {
   var feedURL = 'http://updates.openbazaar.org:5000/update/' + platform + '/' + version;
   autoUpdater.setFeedURL(feedURL);
   mainWindow.webContents.executeJavaScript("console.log('Checking for new versions at " + feedURL + " ...')");
+
+  // Check for updates every hour
   autoUpdater.checkForUpdates();
+  setTimeout(function () {
```

Comment:
This will only check once, one hour after the client starts.

Did you mean to use setInterval instead?